shell/browser/api/electron_api_tray.cc

```diff
@@ -212,11 +212,19 @@ void Tray::SetToolTip(const std::string& tool_tip) {
   tray_icon_->SetToolTip(tool_tip);
 }
 
-void Tray::SetTitle(const std::string& title) {
+void Tray::SetTitle(const std::string& title, gin::Arguments* args) {
```

Comment:
```suggestion
void Tray::SetTitle(const std::string& title, base::Optional<std::string> font_type) {
```

The converter for `Optional<T>` should handle this nicely.